### ISSUE TITLE
Style of text editors is automatically set to style="display: none;"

### ISSUE THREAD

**Issue by RasMakonnen** [tinymce/tinymce#4252]
Do you want to request a feature or report a bug?

BUG

What is the current behavior?

Style of text editors is automatically set to style="display: none;"

If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via fiddle.tinymce.com or similar.

Create text editor like below:
```
<div role="tabpanel" class="tab-pane" id="email-approvers">
@Html.TextAreaFor(m => m.EmailSetting.Template, 
new { id = "email-approvers-editor",@class = "form-control full-width mce email-template", rows = "10", onfocus = "setFocusedElement(this)"})
 </div>
```
In browser:
```
<div role="tabpanel" class="tab-pane" id="email-approvers">
<div id="mceu_520" class="mce-tinymce mce-container mce-panel" hidefocus="1" tabindex="-1" role="application" style="visibility: hidden; border-width: 1px; width: 100%; display: none;">...</div>
<textarea class="form-control full-width mce email-template" cols="20" id="email-approvers-editor" name="EmailSetting.Template" onfocus="setFocusedElement(this)" rows="10" style="display: none;" aria-hidden="true">...</textarea>
</div>

```
What is the expected behavior?

Display element using style="display: block;", according to :

```
.mce-container, .mce-container-body {
    display: block;
}

```

Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?

TinyMCE: 4.7.8
Browser: Chrome Version 64.0.3282.167 (Official Build) (64-bit), Opera Version: 50.0.2762.67 (PGO)

**Comment by fyrkant** [comment]
Have you loaded the skin correctly?

**Comment by RasMakonnen** [comment]
I use default skin, and everything looks good.

**Comment by fyrkant** [comment]
We have now released 4.7.9 that should fix this problem, closing this issue.

**Comment by nikolov-ivan** [comment]
h
